automated install/basic-install.sh

```diff
@@ -88,11 +88,14 @@ if [[ $(command -v apt-get) ]]; then
   # fixes for dependancy differences
   # Debian 7 doesn't have iproute2 use iproute
   ${PKG_MANAGER} install --dry-run iproute2 > /dev/null 2>&1 && IPROUTE_PKG="iproute2" || IPROUTE_PKG="iproute"
-  # Prefer the php metapackage if it's there, fall back on the php5 pacakges
-  ${PKG_MANAGER} install --dry-run php > /dev/null 2>&1 && phpVer="php" || phpVer="php5"
```

Comment:
The `IF && II` always fails a ShellCheck for a logic fault, so that would be rewritten. http://shellcheck.net if you'd like to manually copy and paste to see what the result are.